src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export function Person({ person }) {
+  return (
```

Comment:
it is not convenient to use `person.someProperty`, better restructure the `person` 😉 
`const { prop1, prop2 } = person`